test_programs/noir_test_success/global_eval/src/main.nr

```diff
@@ -0,0 +1,10 @@
+use std::uint128::U128;
+
+// These definitions require `to_be_bits` and `to_le_bits` to be supported at comptime.
+global POW64_A: Field = 2.pow_32(64);
+global POW64_B: Field = (U128::one() << 64).to_integer();
+
+#[test]
+fn test_pow64_equal() {
+    assert_eq(POW64_A, POW64_B);
+}
```

Comment:
Why not test to_be_bits and/or to_le_bits directly (and rename the test)?
That way if the implementation of pow_32 changes we'll still test these same functions.

Reply:
Fair enough, I'll add another test. Just wanted to capture exactly what Jan wanted to do.